contracts/Vault.vy

```diff
@@ -785,6 +785,25 @@ def totalAssets() -> uint256:
     return self._totalAssets()
 
 
+@view
+@internal
+def _calculateLockedProfit() -> uint256:
+    lockedFundsRatio: uint256 = (block.timestamp - self.lastReport) * self.lockedProfitDegration
+
+    if(lockedFundsRatio < DEGREDATION_COEFFICIENT):
+        precisionFactor: uint256 = self.precisionFactor
+        lockedProfit: uint256 = self.lockedProfit
+        return lockedProfit - (
+                precisionFactor
+                * lockedFundsRatio
+                * self.lockedProfit
```

Comment:
we can use `lockedProfit` memory var instead of reading again `self.lockedProfit`